lib/ravelin/errors/api_error.rb

```diff
@@ -0,0 +1,19 @@
+module Ravelin
+  class ApiError < StandardError
+    attr_reader :response, :status, :error, :error_message, :validation_errors
+
+    def initialize(response)
+      @response           = response
+      @status             = response.status
+      @error              = response.body.fetch('Error', nil)
+      @error_message      = response.body.fetch('message', nil)
+      @validation_errors  = response.body.fetch('validationErrors', [])
+    end
+
+    def to_s
```

Comment:
`#to_s` seems to set the exception message to give a more informative output.

For example, `400 - JSON validation error - deviceId is required` has been added to the output below, by defining the `to_s` method in the `InvalidRequestError` class.

```
../ravelin-0.1.0/lib/ravelin/client.rb:48:in `handle_error_response': 400 - JSON validation error - deviceId is required (Ravelin::InvalidRequestError)
	from ../ravelin-0.1.0/lib/ravelin/client.rb:41:in `post'
	from ../ravelin-0.1.0/lib/ravelin/client.rb:17:in `send_event'
	from playground.rb:128:in `<main>'
```